index.d.ts

```diff
@@ -227,7 +227,7 @@ interface ServiceConfig {
 }
 
 interface Gateway {
-  refresh: () => Promise<GraphQLSchema | null>;
+  refresh: (isRetry: boolean) => Promise<GraphQLSchema | null>;
```

Comment:
I would say this is still a breaking change from a type PoV since the current type tests won't pass with this change (we can no longer call `.refresh()` without compilation errors in TypeScript). Can we make the input optional?

Reply:
Sorry, I didn't notice that. The input is optional. I will change it to `refresh: (isRetry?: boolean)`

Reply:
I have changed it. I think the PR should be ready now.